Build/scripts/pgo/pre_pgo.cmd

```diff
@@ -12,6 +12,13 @@
 :: - build (using PGO profile)
 :: - post_pgo.cmd
 
+@echo off
+
+if "%PogoConfig%"=="False" (
+    echo "---- Not a Pogo Config. Skipping step."

```

Comment:
Oh wow I forgot about that. I've been using Git bash too much... bash's echo treats those quotes as parameter delimiters like is the case with other commands.

Anyway I'll remove them.